src/storage.js

```diff
@@ -0,0 +1,122 @@
+//!This file contains functions related to checking for, adding and deleting `.comment` files and `.comments` directories.
+
+/*
+ * c
+ * https://github.com/rumpl/c
+ *
+ * Copyright (c) 2012 Djordje Lukic
+ * Licensed under the MIT license.
+ */
+
+"use strict";
+
+var fs = require("fs"); //FileSystem
+var path = require("path"); //Paths
+var underscore = require("underscore"); //Some functional shit
```

Comment:
Please remove "shit" :)

Reply:
Yeah, sorry, I need to get away from amateurish... stuff.

Reply:
No worries ;)